tutorials/deploy-openproject-database-postgresql/index.mdx

```diff
@@ -6,64 +6,44 @@ content:
   h1: Deploying OpenProject with a Managed Database for PostgreSQL
   paragraph: Learn how to install and configure OpenProject on Ubuntu Focal Fossa 20.04 while integrating a managed database for PostgreSQL instance with your stack.
 tags: OpenProject management panel database PostgreSQL
-steps:
-  - step: OpenProject - Overview
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/deploy-openproject-database-postgresql/#OpenProject-Overview
-  - step: Installing OpenProject
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/deploy-openproject-database-postgresql/#Installing-OpenProject
-  - step: Creating a Managed Database for PostGreSQL
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/deploy-openproject-database-postgresql/#Creating-a-Managed-Database-for-PostGreSQL
-  - step: Configuring the Stack
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/deploy-openproject-database-postgresql/#Configuring-the-Stack
-  - step: Installing a SSL/TLS Certificate
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/deploy-openproject-database-postgresql/#Installing-a-SSL-TLS-Certificate
-  - step: Enabling TLS/SSL
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/deploy-openproject-database-postgresql/#Enabling-TLS/SSL
-  - step: Accessing your OpenProject Dashboard
-    type: HowToStep
-    url: https://www.scaleway.com/en/tutorials/deploy-openproject-database-postgresql/#Accessing-your-OpenProject-Dashboard
 categories:
   - compute
   - storage
 image: /assets/scaleway_openproject.png
 dates:
-  validation: 2021-05-10
+  validation: 2022-03-03
   posted: 2020-07-23
 ---
 
 ## OpenProject - Overview
 
-[OpenProject](https://www.openproject.org/) is an open source project management software. Available in more than 30 languages, it provides tools and features to assist teams with communication, collaboration and navigating the lifecycle of a project.
+[OpenProject](https://www.openproject.org/) is open source project management software. Available in more than 30 languages, it provides tools and features to assist teams with communication, collaboration and navigating the lifecycle of a project.
```

Comment:
Is there a particular reason that you deleted the "an" in the original version?

Reply:
because "a software" isn't technically gramatically correct, but I am splitting hairs ;-)

Reply:
Ahh okay I didn't know that hehe, then all good, you get the 🦭